### PR TITLE
included BTrace script for profiling Java KRun

### PR DESCRIPTION
**Pull request by yilongli**

### CODE REVIEW

File: src/javasources/KTool/src/org/kframework/backend/java/profile/JavaKRunTrace.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
+package org.kframework.backend.java.profile;
```
Comment (dwightguth) [inline src/javasources/KTool/src/org/kframework/backend/java/profile/JavaKRunTrace.java]: Did you write this file by hand or was it generated somehow?

File: src/javasources/KTool/src/org/kframework/backend/java/profile/package-info.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/**
```
Comment (dwightguth) [inline src/javasources/KTool/src/org/kframework/backend/java/profile/package-info.java]: What is this file for?

**Comment by dwightguth** [comment]
Let's put the new btrace Java file in src/main/btrace/org/kframework/backend/java/profile. That's where it would go under a Maven structure. You can then do whatever fancy stuff you need to do in your local copy to compile it properly.

File: src/javasources/KTool/src/org/kframework/backend/java/profile/JavaKRunTrace.java (hunk 1 above)
Comment (yilongli) [inline src/javasources/KTool/src/org/kframework/backend/java/profile/JavaKRunTrace.java]: Unfortunately, by hand:)

File: src/javasources/KTool/src/org/kframework/backend/java/profile/package-info.java (hunk 2 above)
Comment (yilongli) [inline src/javasources/KTool/src/org/kframework/backend/java/profile/package-info.java]: It's for package level javadoc.

**Comment by yilongli** [comment]
Sounds good.

**Comment by dwightguth** [comment]
Sorry, I was unclear. I mean k/src/main/btrace, not k/src/javasources/KTool/src/main/btrace. Can I see a quick code review which fixes this? Should take like 10 minutes tops.

**Comment by yilongli** [comment]
I issued a new pull request for this fix: https://github.com/kframework/k/pull/311

**Pull request merged**